src/MusicStore/Invoke-Crossgen.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference = "Stop"
 
 $lib_paths = @()
 
-$excludes = @("MusicStore.dll", "Microsoft.AspNetCore.Diagnostics.dll", "Microsoft.AspNetCore.Hosting.Abstractions.dll", "Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.dll", "Microsoft.AspNetCore.Hosting.dll", "Microsoft.AspNetCore.WebUtilities.dll", "Microsoft.DiaSymReader.Native.amd64.dll", "Microsoft.DiaSymReader.Native.x86.dll")
```

Comment:
The main issue with these appears was a mismatch between the assembly versions of `System.Diagnostics` which appears to have gone away in netstandard2.0. 👍 

The `Microsoft.DiaSymReader.Native.*.dll` dlls aren't here anymore so they don't need to be excluded.

---

With these changes, the same set of binaries are going to be crossgened as we expect in local dev scenarios in 2.0.0.